package.json

```diff
@@ -18,7 +18,7 @@
     "scripts:release-post": "cross-env NODE_NO_WARNINGS=1 node scripts/release-post/index.mjs",
     "scripts:generate-next-data": "cross-env NODE_NO_WARNINGS=1 node scripts/generate-next-data/index.mjs",
     "preserve": "npm run scripts:generate-next-data",
-    "serve": "cross-env NODE_NO_WARNINGS=1 next dev",
+    "serve": "cross-env NODE_NO_WARNINGS=1 next dev --turbo",
```

Comment:
why not use "dev" it's nextjs default value

Reply:
Wdym?

Reply:
I mean on when you use `create-next-app`, `next dev` is under script "dev". and on our codebase we use "serve". So I propose to use dev like in nextjs convention

Reply:
This is just semantics. serve is the name of the command weve been using for a long period of time. Why the fuck would we change that lol.

And what does cra (create react app) has to do with this repo?

Reply:
>This is just semantics. serve is the name of the command weve been using for a long period of time. Why the fuck would we change that lol.

because that an false name. `next dev` it's command for development not for serving 

> And what does cra (create react app) has to do with this repo?

I'm speaking about `CNA`

Reply:
False name? Give me a break.

Reply:
Augustin has good points, especially considering looking at the site through the eyes of new contributors. I also want to mention that profanity here doesn't seem necessary.

- https://nextjs.org/docs/app/api-reference/next-cli#development straight up uses `dev`. Precedent isn't always a valid  reason to keep something. 
- He is referring to https://nextjs.org/docs/pages/api-reference/create-next-app - a first party tool maintained by Next.

If this is something anyone feels strongly about, a new issue should be created so the community can discuss it in earnest there, and not as a tangent part of an unrelated PR.

Reply:
I am just tired of Augustin insisting by himself on the same point over and over. I genuinely don't care in this case if other projects call the command "dev"; That by itself isn't an argument. We have the liberty of doing things the way we want.

We use the same command "serve" for so many years, and so far I haven't seen a single issue being opened of people complaining about the command. Existing collaborators are familiar with it, new ones easily learn what the command is.

The command is not semantically wrong because "next dev" technically does start a process that serves the website.

My usage of profanity words is uncalled for, but I am just tired of having random argumentations about the same thing over and over that are completely off-topic.

Open an issue if any of you have strong arguments about this;

Reply:
Just put yourself in my shoes, would you not lose your patience if someone keeps questioning your decisions over and over non-stop to a point that it becomes annoying? Even more if the questioning is about a repetitive topic or something that does not change a single thing?

What are we expecting to gain here by changing a command people are familiar with already with no apparent reason besides of "hey I think a few other projects use this word"...

If y'all want the change, feel free to open a PR or an issue and do it. I wouldnt block it. But still perhaps, an unnecessary change.